src/extension.ts

```diff
@@ -594,6 +594,8 @@ export function activate(context: vscode.ExtensionContext) {
     function getChannelPath(): string {
         if (vscode.env.appName.indexOf("Insiders") > 0) {
             return "Code - Insiders";
+        } else if (vscode.env.appName.indexOf("OSS") > 0) {
```

Comment:
I would make a small change. Instead of another `if` statement, it would be better to use:

```typescript
    function getChannelPath(): string {
      return vscode.env.appName.replace("Visual Studio ", "");
    }
```

Reply:
Don't worry if you can't update the PR right now. I can do it myself before merging. I was just _playing_ with code-review 😀

Reply:
Sure thing, I'll update it right now. I've never touched TypeScript, so I didn't want to take any chances and went with something simple that I knew it would work. This is better, though.

Reply:
That's great. Btw, my first TypeScript experience was also with VS Code extensions 😄